test/evergreen.yml

```diff
@@ -6015,6 +6026,246 @@ tasks:
         vars:
           test_path: bench/wt2853_perf/wt2853_perf
 
+  - name: "workgen-test-compress_ratio"
```

Comment:
I'll review. I wanted to use the generate.tasks but that is a longer term item.

Reply:
This is slightly different because there is a variable change per instance. I'll see if I can make it related to the name of the task which could make it reusable.

Reply:
Cleaned this up, I think its better but not perfect.

Reply:
Thanks!